plugins/openquakeqgiswebservices/openquakeqgiswebservices.py

```diff
@@ -0,0 +1,65 @@
+import json
+from qgis.server import QgsService
+
+
+class OpenQuakeQGISWebServices(QgsService):
```

Comment:
@nastasi-oq was suggesting to keep the plugin more generic, with no references to OQ.

It's true that the plugins has nothing customized for OQ.

So it would be good to rename the classes and the service (i.e. to something like EWMS, Extended WMS).

Reply:
I agree. I will do it.